proposals/76:vaults-auctions/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "agoricProposal": {
+    "type": "/agoric.swingset.CoreEvalProposal",
+    "releaseNotes": "https://github.com/Agoric/agoric-sdk/releases/tag/agoric-upgrade-16av",
```

Comment:
the build-proposals job is failing because this trailing comma is invalid JSON.

maybe we should add a "lint" job? it could report if `yarn doctor` reported or changed anything and also do eslint style linting on files.